packages/manager/apps/telecom/src/components/sidebar/sidebar.config.js

```diff
@@ -103,7 +104,11 @@ angular
         }
 
         // add sidebar SMS item
-        if (get(count, 'sms', beta)) {
+        if (
+          (get(count, 'sms') &&
+            featuresAvailabilities.isFeatureAvailable('sms')) ||
+          beta
+        ) {
```

Comment:
```suggestion
        if (featuresAvailabilities.isFeatureAvailable('sms') && 
          (get(count, 'sms') || beta)
        ) {
```

I think we want to display "SMS" in the sidebar if we have sms services or if we are in beta and this and only if the feature is enabled.

Reply:
Sounds good for me